csharp/tutorials/patterns/start/toll-calculator/Program.cs

```diff
@@ -3,126 +3,119 @@
 using ConsumerVehicleRegistration;
 using LiveryRegistration;
 
-namespace toll_calculator
+using toll_calculator;
```

Comment:
This looks like a copy / paste error:

```suggestion
namespace toll_calculator;
```